src/Common/Scheduler/Workload/WorkloadEntityDiskStorage.h

```diff
@@ -39,6 +44,10 @@ class WorkloadEntityDiskStorage : public WorkloadEntityStorageBase
 
     String dir_path;
     std::atomic<bool> entities_loaded = false;
+    
+    /// Config-based entities storage
+    std::shared_ptr<WorkloadEntityConfigStorage> config_storage;
+    scope_guard config_subscription;
```

Comment:
The subscription turned out to be unnecessary, as we will do updates through loadEntities().